ihatemoney/models.py

```diff
@@ -80,13 +81,17 @@ def members_stats(self):
             {
                 "member": member,
                 "paid": sum(
-                    [bill.amount for bill in self.get_member_bills(member.id).all()]
+                    [
+                        bill.amount
+                        for bill in self.get_member_bills(member.id).all()
+                        if bill.type != BillTypes.settlement.value
```

Comment:
I think I don't see the value of having a type for this. We probably could use strings instead (or constants)